Jenkinsfile

```diff
@@ -3,88 +3,51 @@
 // https://github.com/feedhenry/fh-pipeline-library
 @Library('fh-pipeline-library') _
 
-def repositoryName = "metrics-apb"
-def projectName = "test-${repositoryName}-${currentBuild.number}-${currentBuild.startTimeInMillis}"
-
 stage('Trust') {
     enforceTrustedApproval('aerogear')
 }
 
-node ('apb-test') {
+node ("ocp-slave") { 
+    def apb_container_id, pod_container_id, apb_pod_output
+    def apb_version = "sprint147.2"
+    
+    stage('Cleanup') {
+        deleteDir()
```

Comment:
@psturc other than deleting the dir, is there any other cleanup required?
Before this change, the openshift project was deleted as a cleanup step. Is that not required now?

Reply:
`apb test` command takes care of cleanup of the project whether it succeeded or not, so deleting working directory only is fine.